src/main/java/Employee.java

```diff
@@ -0,0 +1,32 @@
+public class Employee {
+    private String status;
```

Comment:
please take a look at the lombok plugin, it has a set of convenient annotations that allow to keep the code cleaner. In this case @Data annotation seems to be a good fit.